logos/logos-orchestrator/tests/unit/capacity/test_calibration_lane_guard.py

```
@@ -0,0 +1,372 @@
+    registry, _session, _ = _registry_answering_start(reply={"ok": False, "error": "node is in a degraded state"})
+
+    asyncio.run(registry.send_command(1, "start_calibration_session"))
+
+    assert registry.is_calibrating(1) is False
+
+
+def test_a_refused_start_keeps_an_already_running_session_excluded():
+    """The worker refuses a second start while one is in progress. Undoing must
+    restore the previous value, not clear it — clearing would hand the running
+    session's VRAM to the planner."""
+    registry, session, _ = _registry_answering_start(
```

Ruff (0.16.1)

[warning] Unpacked variable `session` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)